test/parallel/test-dgram-setTTL.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const dgram = require('dgram');
+const socket = dgram.createSocket('udp4');
+
+socket.bind(common.PORT);
+socket.on('listening', function() {
```

Comment:
I don't believe it would make a difference if the callback is not invoked, since the bind will keep the process alive (or throw if it fails). I believe `common.mustCall` is called in the `exit` event, which wouldn't happen either if the callback isn't called.